### PR TITLE
Added meraki web link- get link and create link

### PR DESCRIPTION
**Pull request by Poonam-Singh-Bagh**
Added meraki web link- get link and create link

### CODE REVIEW

File: src/components/Dashboard /PartnerData/index.js [hunk 1]
```diff
@@ -115,6 +105,7 @@ function PartnerDashboard() {
               <th>Partner's Name</th>
               <th>Number of students</th>
               <th>Meraki Link</th>
```
Comment (pysaquib) [inline src/components/Dashboard /PartnerData/index.js]: Rename the table header to `Meraki - Android Link`

File: src/components/Dashboard /PartnerData/index.js [hunk 2]
```diff
@@ -115,6 +105,7 @@ function PartnerDashboard() {
...
+              <th>Meraki Web Link</th>
```
Comment (pysaquib) [inline src/components/Dashboard /PartnerData/index.js]: Rename the table header to `Meraki - Web Link`

File: src/components/Dashboard /PartnerData/index.js [hunk 3]
```diff
@@ -146,7 +137,28 @@ function PartnerDashboard() {
                     <td data-column="Meraki Link">
                       <div
                         className="create-link"
-                        onClick={() => createMerakiLink(item.id)}
+                        onClick={() => createMerakiLink(item.id, "android")}
```
Comment (pysaquib) [inline src/components/Dashboard /PartnerData/index.js:140]: Is the equivalent of this for "web" also done?

Comment (Poonam-Singh-Bagh) [reply src/components/Dashboard /PartnerData/index.js:140]: Sorry. Didn't get you.

**Pull request merged**